org.guitarix.Guitarix.json

```diff
@@ -23,8 +23,8 @@
         "--share=network",
         /* Allow other instances to see lockfiles */
         "--env=TMPDIR=/var/tmp",
-        "--env=LV2_PATH=/app/extensions/Plugins/lv2",
-        "--env=LADSPA_PATH=/app/extensions/Plugins/ladspa"
+        "--env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2:/app/lib/lv2",
```

Comment:
just like in other apps, remember that this needs a wrapper script

Reply:
yeah I have to fix that.